packages/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,178 @@
+      (current > panels - 2 && index > panels - 4)
+    )
+  }
+
+  checkForEllipsis = index => {
+    const { current } = this.state
+    const { panels } = this.state
+    // Check if we should render an ellipsis
+    // if index is less than two or greater than two of current
+    // if current is less than three and index is five
+    // and the inverse if current is within two of total and index is total minus 5
+    return (
+      index === current - 2 ||
+      index === current + 2 ||
+      (current < 3 && index === 5) ||
+      (current > panels - 2 && index === panels - 5)
+    )
+  }
+
+  mapTabs = () => {
+    const goToText = this.props.language === 'french' ? 'Aller au panneau n°' : 'Go to panel number'
+    return this.props.children.map((item, i) => {
+      const index = i + 1
+      let { current } = this.state
+      current = parseInt(current, 10) || 0
+      if (current === index) {
+        return (
+          <li key={hash(`${i}-1`)} className={styles.current}>
+            {index}
+          </li>
+        )
+      }
+      if (this.checkForRegularTabs(index)) {
+        return (
+          <li key={hash(`${i}-3`)} className={styles.regular}>
+            <button
+              value={index}
+              onClick={e => this.handleClick(e)}
+              aria-label={`${goToText} ${index}`}
+            >
+              {index}
+            </button>
+          </li>
+        )
+      }
+      if (this.checkForEllipsis(index)) {
+        return (
+          <li key={hash(`${i}-4`)} className={styles.ellipsis}>
+            ...
+          </li>
+        )
+      }
+
+      return null
+    })
+  }
+
+  handleClick = e => {
+    e.preventDefault()
+    const value = parseInt(e.currentTarget.value, 10) || 0
+    if (value > this.state.panels || value < 1) {
+      return null
+    }
+    return this.setState({ current: value }, this.handleButtonState)
+  }
+
+  handleButtonState = () => {
+    if (this.state.current !== 1) {
+      this.setState({ showPrevious: true })
+    } else {
+      this.setState({ showPrevious: false })
+    }
+    if (this.state.current !== this.state.panels) {
+      this.setState({ showNext: true })
+    } else {
+      this.setState({ showNext: false })
+    }
+  }
+
+  render() {
+    const { children, language, ...rest } = this.props
+    const { current } = this.state
+    const increaseNumber = parseInt(current + 1, 10)
+    const decreaseNumber = parseInt(current - 1, 10)
+    const PreviousText = language !== 'french' ? 'Previous' : 'Précédent'
+    const NextText = language !== 'french' ? 'Next' : 'Suivant'
+    const PreviousLabel =
+      language !== 'french' ? 'Go to previous panel' : 'Aller au panneau précédent'
+    const NextLabel = language !== 'french' ? 'Go to next panel' : 'Aller au prochain panneau'
+    return (
+      <div {...safeRest(rest)} className={styles.container}>
+        <Panels {...rest}>{children[current - 1] && children[current - 1]}</Panels>
```

Comment:
Can `{children[current - 1] && children[current - 1]}` just be `{children[current - 1]}`?